chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -896,4 +899,4 @@ Zotero.Translate.ItemGetter.prototype = {
 		return obj;
 	}
 }
-Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
\ No newline at end of file
+Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
```

Comment:
I edited the file online on github, but I've done a local checkout and tried both `git checkout -p` and `git checkout -p chrome/content/zotero/xpcom/translation/translate_item.js`, and in both cases it reports back `No changes.`